handler/consumer/api/src/main/java/io/knotx/fragments/handler/consumer/api/FragmentExecutionLogConsumer.java

```diff
@@ -13,26 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.knotx.fragments.handler.api.consumer;
+package io.knotx.fragments.handler.consumer.api;
 
 import io.knotx.fragments.engine.api.FragmentEvent;
 import io.knotx.fragments.handler.api.metadata.TasksMetadata;
+import io.knotx.fragments.handler.consumer.api.model.FragmentExecutionLog;
 import io.knotx.server.api.context.ClientRequest;
 import java.util.List;
 
 /**
- * Fragment event consumer receives {@link FragmentEvent} when {@link io.knotx.fragments.engine.api.Task}
- * evaluation ends. It can share this information with some external tools or even modify fragment.
+ * Fragment execution log consumer receives {@link FragmentExecutionLog} after the {@link
+ * io.knotx.fragments.engine.api.Task} evaluation.
  */
-public interface FragmentEventsConsumer {
+public interface FragmentExecutionLogConsumer {
 
   /**
-   * Gets a list of processed and unprocessed fragments.
+   * Gets a list of processed and unprocessed fragments (execution logs).
    *
-   * @param clientRequest - client request
-   * @param fragmentEvents - all fragment events
-   * @param tasksMetadata - mapping from fragment id to associated task's metadata
+   * @param request - original request data
+   * @param executions - list of fragment execution logs
    */
-  void accept(ClientRequest clientRequest, List<FragmentEvent> fragmentEvents, TasksMetadata tasksMetadata);
+  void accept(ClientRequest request, List<FragmentExecutionLog> executions);
```

Comment:
Just wondering if this is enough information for any generic consumer we expect to appear. ClientRequest now serves as activation condition for a consumer - could there be any more conditions or alternative usages?
In any case, it really seems to be sufficient at the moment.

Reply:
Fair point. If necessery, we can always `pump up` the interface later.